project-3/jobs.hpp

```diff
@@ -3,33 +3,29 @@
 #include <iostream>
 #include <pthread.h>
 #include <cassert>
-struct Stat{
-	std::string sellerType;
-	int waitTime;
-	int responseTime;
-	int turnaroundTime;
-};
-//class for your basic Job;
-class Job{	
-	public:
-	//public variables (yay?)
-	std::string name;
-	int arrivalTime, serviceTime, priority;
-	
-	//constructors
-	Job();
-	Job(int a, int s, int p, std::string name);
-    
-		
-	//getter stuff
-	std::string getName(){return name;}
-	int getArr(){return arrivalTime;}
-	const int getArr() const{return arrivalTime;}
-	int getServ(){return serviceTime;}
-	const int getServ() const{return serviceTime;}  
-	 
-	
-};
+#include "linked.hpp"
+
+////class for your basic Job;
```

Comment:
This could have just been renamed something else rather than commented out, not really a compliant but just a suggestion.